hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -2213,4 +2213,43 @@ private void createLinkBucket(String linkVolume, String linkBucket,
     OzoneVolume ozoneVolume = objectStore.getVolume(linkVolume);
     ozoneVolume.createBucket(linkBucket, builder.build());
   }
+
+  @Test
+  public void testSnapshotRead() throws Exception {
```

Comment:
Just one question, this new UT is not guaranteed to repro the issue, correct?

In other words, without the fix at least this UT will be flaky?

Reply:
yes.